src/lib/media-import/status.ts

```diff
@@ -306,12 +308,16 @@ Downloading errors details from ${ fileErrorsUrl }
 	}
 
 	async function promptFailureDetailsDownload( fileErrorsUrl: string ) {
-		const failureDetails = await prompt( {
-			type: 'confirm',
-			name: 'download',
-			message:
-				'Download import errors report now? (Report will be downloadable for up to 7 days from the completion of the import)',
-		} );
+		const failureDetails = ! saveErrorLog
+			? await prompt( {
+					type: 'confirm',
+					name: 'download',
+					message:
+						'Download import errors report now? (Report will be downloadable for up to 7 days from the completion of the import)',
+			  } )
+			: {
+					download: true,
+			  };
```

Comment:
If `--saveErrorLog no` is specified, we still get this:
```
? Download import errors report now? (Report will be downloadable for up to 7 days from the completion of the import) (y/N)
```

I think it should not ask for user-input on whether to download the logs or not, since the user has already answered that.

Reply:
We have to maintain three states of `saveErrorLog`: `true`, `false` and `undefined`.

Reply:
True - we'll need to make it not-a-boolean, then, since it would have three possible values - yes, no, or prompt.

Reply:
Correct.

Reply:
I pushed up a change that should address this. I'm testing it locally now.

Reply:
Because this is a string type now and not a boolean, I don't think we'll be able to allow users to pass it without specifying a value and have it count as "true" unless we change the default. Therefore, it will need to be invoked with `--saveErrorLog=true` or `--saveErrorLog=yes` in order to bypass, and `prompt` will be the default, and users can pass `false` or `no` to not prompt and have it not save the log file. I'm pushing up this change now.

Reply:
This is working well in my tests so far.

Reply:
This seems to solve the issue:

```diff
diff --git a/src/bin/vip-import-media.js b/src/bin/vip-import-media.js
index 39f23e1a..ceffe1b8 100755
--- a/src/bin/vip-import-media.js
+++ b/src/bin/vip-import-media.js
@@ -98,7 +98,7 @@ Are you sure you want to import the contents of the url?
 		'Export any file errors encountered to a JSON file instead of a plain text file',
 		false
 	)
-	.option( 'saveErrorLog', 'Download file-error logs without prompting', 'prompt' )
+	.option( 'saveErrorLog', 'Download file-error logs without prompting' )
 	.option( 'overwriteExistingFiles', 'Overwrite any existing files', false )
 	.option( 'importIntermediateImages', 'Import intermediate image files', false )
 	.examples( examples )
diff --git a/src/lib/cli/command.js b/src/lib/cli/command.js
index 1863147a..68c11420 100644
--- a/src/lib/cli/command.js
+++ b/src/lib/cli/command.js
@@ -476,6 +476,8 @@ args.argv = async function ( argv, cb ) {
 					value: options.exportFileErrorsToJson ? '✅ Yes' : `${ chalk.red( 'x' ) } No`,
 				} );

+				options.saveErrorLog =
+					options.saveErrorLog === undefined ? 'prompt' : `${ options.saveErrorLog }`;
 				info.push( {
 					key: 'Download file-error logs?',
 					value: options.saveErrorLog,
```

Reply:
That block of code only runs if the user doesn't specify `--force`. I'd imagine that the most common use case is in combination with `--force` to allow commands to be chained without user input. We could move negotiation of that option up to above the if conditional for force, though. I'll take a pass at this.

Reply:
I pushed up a change that should allow for this. There are a few other places where option values are being set only if the force flag is not set which you may want to also move up to this new switch statement: [here](https://github.com/Automattic/vip-cli/blob/trunk/src/lib/cli/command.js#L392-L395), [here](https://github.com/Automattic/vip-cli/blob/trunk/src/lib/cli/command.js#L452-L455), [here](https://github.com/Automattic/vip-cli/blob/trunk/src/lib/cli/command.js#L461-L464), and [here](https://github.com/Automattic/vip-cli/blob/trunk/src/lib/cli/command.js#L470-L473).